test/build.bats

```diff
@@ -164,7 +164,7 @@ OUT
   stub_make_install
 
   run_inline_definition <<DEF
-install_package "ruby-2.0.0" "http://ruby-lang.org/ruby/2.0/ruby-2.0.0.tar.gz"
+install_package "ruby-2.0.0" "https://cache.ruby-lang.org/pub/ruby/2.0/ruby-2.0.0.tar.gz"
```

Comment:
This change to tests wasn't necessary. The test can continue to keep using HTTP resources since they are fake anway. It's not like the tests are "insecure".